core/shared/platform/include/platform_api_vmcore.h

```diff
@@ -66,6 +66,14 @@ os_vprintf(const char *format, va_list ap);
 uint64
 os_time_get_boot_microsecond(void);
 
+/* Get current resolution of clock id */
+int
+os_clock_res_get(bh_clock_id_t clock_id, uint64 *resolution);
```

Comment:
we already have the declaration in platform_common.h so the declaration should probably be removed from this file.